src/components/Main.css

```diff
@@ -42,3 +40,8 @@
 #root main > div > div {
   min-height: 100% !important;
 }
+
+.ReactModal__Content:focus-visible {
+  outline: none;
+  box-shadow: inset 0 0 0 2px #e58900, inset 0 0 0 3px #fff;
+}
```

Comment:
Should we use a personalized style-guide class in place of this one (`ReactModal__Content`)?

Reply:
@GusBaamonde I like this aproach, what do you think about this?

Reply:
This could be an improvement to our modals but I don´t know if this is related to accessibility.

Reply:
Talking with @GusBaamonde we decide move this change to the style guide.

Reply:
This change was'n necessary using the _react-modal_ prop `shouldFocusAfterRender=false`